### ISSUE TITLE
Training QA tester on screen reader?

### ISSUE THREAD

**Issue by emily-phet** [phetsims/tasks#618]
Can one (or more) of the QA testers start doing some basic testing of the accessible prototypes using screen readers (JAWS, NVDA, and Voiceover only)? This would likely require a meeting with @jessegreenberg to introduce them to how screen readers work, where to find lists of common hotkeys (e.g., how to turn the reader on/off and navigate a page) and some expected behavior of buttons, sliders, etc. in a sim. 

For each sim that needs to be tested, there would likely be a set of specific interactions we would want them to look at, and would let them know these in advance.

**Comment by phet-steele** [comment]
@emily-phet @ariel-phet @jessegreenberg oh yes please!

**Comment by ariel-phet** [comment]
@phet-steele this is definitely one for you.

Let also in this issue begin developing a "screen reader testing checklist" that we can eventually put in phet-info (once you learn more)

**Comment by emily-phet** [comment]
@jessegreenberg - could you introduce @phet-steele to some screen readers, and the types of information we would care about from testing?

**Comment by jessegreenberg** [comment]
Absolutely, @phet-steele and I will be meeting today to talk screen readers.

**Comment by jessegreenberg** [comment]
It's a good idea to create a check list for testing accessible content.  Brainstorming a few things that could be on that list.

- [ ] Does every element have a label?  (For instance, is it clear what a specific button does?)
- [ ] Do descriptions correctly reflect the state of the simulation, and do they change when necessary?
- [ ] Can you tab to all interactive components?
- [ ] Do alerts correctly describe everything on screen and at the correct time?
- [ ] Does the focus highlight move correctly with keyboard navigation?
- [ ] After inspecting the HTML, does it seem like everything can be found by the screen reader?
    - requires dev tools, maybe we can invest in a tool to 'see' entire document, kind of like our other iframe prototype?
- [ ] Does tab touch every navigable element?
- [ ] Do screen reader navigation strategies work on appropriate elements for the sim?
  - Does 'L' pull find the next list?  Does 'B' find the next button? etc...
- [ ] Does keyboard navigation work when the screen reader is turned off?

**Comment by emily-phet** [comment]
@jessegreenberg - I felt inspired, and made a Google doc with a draft of a checklist: https://docs.google.com/document/d/1jYx3wxNRxHHpMigxSFVvCRVFdB5-E-3_xbj9UI9uvWE/edit?usp=sharing

Essentially, it seems like there are multiple categories of to-dos in your checklist, so listed these out. Does this look helpful? Feel free to add/use/copy/steal whatever - doc should be editable.

**Comment by jessegreenberg** [comment]
Thanks @emily-phet, that looks great!  Perhaps we can pass along to IDRC folks, I am sure they would have some great ideas to add here.  @phet-steele might also have some recommendations for the list too.

@phet-steele and @jessegreenberg met on 6/17/16 and we went over how to read web documents and PhET sims with a screen reader.  I am sure there will be new items as we continue, but that is why the document above is so helpful.

**Comment by jessegreenberg** [comment]
Given the above comment, I think we can close this issue.  Document created by @emily-phet has been linked in an issue in phetsims/a11y-issues and will be finished in that repo.
